assets/xml/objects/gameplay_keep.xml

```diff
@@ -1,369 +1,951 @@
+        <Texture Name="gEffectLightning8Tex" OutName="effect_lightning_8" Format="i8" Width="16" Height="96" Offset="0x0002C930"/>
+        <DList Name="gEffectLightningDL" Offset="0x0002CF30"/>
+        <Texture Name="gEffectEnemyDeathFlame1Tex" OutName="effect_enemy_death_flame_1" Format="i8" Width="32" Height="64" Offset="0x0002CFE0"/>
+        <Texture Name="gEffectEnemyDeathFlame2Tex" OutName="effect_enemy_death_flame_2" Format="i8" Width="32" Height="64" Offset="0x0002D7E0"/>
+        <Texture Name="gEffectEnemyDeathFlame3Tex" OutName="effect_enemy_death_flame_3" Format="i8" Width="32" Height="64" Offset="0x0002DFE0"/>
+        <Texture Name="gEffectEnemyDeathFlame4Tex" OutName="effect_enemy_death_flame_4" Format="i8" Width="32" Height="64" Offset="0x0002E7E0"/>
+        <Texture Name="gEffectEnemyDeathFlame5Tex" OutName="effect_enemy_death_flame_5" Format="i8" Width="32" Height="64" Offset="0x0002EFE0"/>
+        <Texture Name="gEffectEnemyDeathFlame6Tex" OutName="effect_enemy_death_flame_6" Format="i8" Width="32" Height="64" Offset="0x0002F7E0"/>
+        <Texture Name="gEffectEnemyDeathFlame7Tex" OutName="effect_enemy_death_flame_7" Format="i8" Width="32" Height="64" Offset="0x0002FFE0"/>
+        <Texture Name="gEffectEnemyDeathFlame8Tex" OutName="effect_enemy_death_flame_8" Format="i8" Width="32" Height="64" Offset="0x000307E0"/>
+        <Texture Name="gEffectEnemyDeathFlame9Tex" OutName="effect_enemy_death_flame_9" Format="i8" Width="32" Height="64" Offset="0x00030FE0"/>
+        <Texture Name="gEffectEnemyDeathFlame10Tex" OutName="effect_enemy_death_flame_10" Format="i8" Width="32" Height="64" Offset="0x000317E0"/>
+        <DList Name="gEffectEnemyDeathFlameDL" Offset="0x00031FE0"/>
+        <Texture Name="gEffectUnknown9Tex" OutName="effect_unknown_9" Format="i8" Width="32" Height="32" Offset="0x00032090"/>
+        <Texture Name="gEffectUnknown10Tex" OutName="effect_unknown_10" Format="i8" Width="32" Height="32" Offset="0x00032490"/>
+        <Texture Name="gEffectUnknown11Tex" OutName="effect_unknown_11" Format="i8" Width="32" Height="32" Offset="0x00032890"/>
+        <Texture Name="gEffectUnknown12Tex" OutName="effect_unknown_12" Format="i8" Width="32" Height="32" Offset="0x00032C90"/>
+        <Texture Name="gUnknownWoodBoardTex" OutName="unknown_wood_board" Format="i8" Width="32" Height="32" Offset="0x00033090"/>
+        <DList Name="gEffectIceFragment1DL" Offset="0x00033720"/>
+        <DList Name="gEffectIceFragment2DL" Offset="0x00033818"/>
+        <DList Name="gEffectIceFragment3DL" Offset="0x00033EE0"/>
+        <Animation Name="gBugCrawlAnim" Offset="0x000341FC"/>
+        <DList Name="gBugLimb1DL" Offset="0x00034210"/>
+        <DList Name="gBugLimb1WrapperDL" Offset="0x000342B0"/>
+        <DList Name="gBugLimb1WrapperWrapperDL" Offset="0x000342C0"/>
+        <DList Name="gBugLimb2DL" Offset="0x00034300"/>
```

Comment:
yes i had to do this before because zapd wasn't decompiling all the displaylists, but i will try again, i can't get zapd to update though with subrepo

Reply:
I think this is still unresolved?

Reply:
he explained some issue to me where the bug display list references other display lists via a DE command that zap isnt catching or something.. idk, sounds like a zap issue we need to investigate separately, unless i misunderstood what zel meant

Reply:
Yeah exactly.